spec/std/file_utils_spec.cr

```diff
@@ -97,138 +97,102 @@ describe "FileUtils" do
 
   describe "touch" do
     it "creates file if it doesn't exists" do
-      filename = File.join(__DIR__, "data/test_touch.txt")
-      begin
-        File.exists?(filename).should be_false
```

Comment:
you removed this check, is it intentional?

Reply:
`with_tempfiles` should guarantee that the path does not exist, don't need to check that again.